kolibri/utils/cli.py

```diff
@@ -668,6 +670,12 @@ def main(args=None):  # noqa: max-complexity=13
         stop()
         return
 
+    if arguments['stop-clean']:
+        call_command("clearsessions")
+        call_command("vacuumsqlite")  # empty WAL files into DB first
+        stop()
+        return
```

Comment:
The order here: Is it safe to call these commands while Kolibri is still running (before it's stopped)? And looking into the :crystal_ball: will this change?

Reply:
yeah that's a good point